modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -1376,3 +1376,13 @@
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/glasses.dmi'
 	icon_state = "supergarsred"
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/eyes.dmi'
+
+//Donation reward for Konstyantyn
+	/obj/item/clothing/accessory/badge/holo/jade
+    name = "jade holobadge"
+    desc = "A strangely green holobadge. 'Lieutenant Uriah' is stamped onto it, above the letters JS."
+    icon = 'modular_skyrat/master_files/icons/donator/obj/custom.dmi'
+    worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/custom_w.dmi'
+    icon_state = "greenbadge"
+    slot_flags = ITEM_SLOT_NECK
```

Comment:
```suggestion
```
nitpick, but this might not be needed considering that this inherits the slot_flags from the standard badge which already has the slot_flags set to item_slot_neck.

It doesn't matter much either way though.